### DISCUSSION TITLE
Could you, please, provide some example of using Bootstrap's Carousel on mobile devices (touch)?

### DISCUSSION THREAD

**Discussion by tesicg** [twbs/bootstrap#32989] [Q&A] [answered]
I mean on example where we have Bootstrap's Carousel that works on larger screens where we use mouse in order to go next/previous and at the same time to work on smaller screens (mobile devices) by using fingertip to swipe slides in the same next/previous manner.

**Comment by coliff** [comment, marked as answer]
The carousels work nicely on mobile (with swipe support) already:
https://getbootstrap.com/docs/5.0/examples/carousel/
https://getbootstrap.com/docs/4.6/examples/carousel/

**Comment by tesicg** [comment]
Ok. But, when I'm trying to swipe it in Chrome DevTools with gray circle, it doesn't work. Should it work?

And what about the code? Where can I take a look at the code behind Carousel with "touch" option?

**Reply by coliff** [reply]
It’s best to test on a real mobile device. You can do view source to get the code for it

**Reply by tesicg** [reply]
Ok. I did it and I didn't see any special code related to mobile devices, such as "touch" option. Does it mean that Bootstrap's Carousel works by default in that way that is I don't need to setup anything special?

**Reply by XhmikosR** [reply]
It's how Browser emulation works on Chrome IIRC. And yeah, it should work on mobile without anything from your side.
